examples/searchcommands_app/package/default/logging.conf

```diff
@@ -30,9 +30,9 @@ propagate = 0     ; Default: 1
 
 [logger_GenerateTextCommand]
 qualname = GenerateTextCommand
-level = NOTSET    ; Default: WARNING
-handlers = app    ; Default: stderr
-propagate = 0     ; Default: 1
+level = NOTSET          ; Default: WARNING
+handlers = splunklib    ; Default: stderr
+propagate = 0           ; Default: 1
```

Comment:
should also update the `logger_GenerateHelloCommand` stanza

Reply:
done